### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by janzd** [HCIILAB/Scene-Text-Removal#18]
Are the default settings of parameters such as learning rate the parameters that you used for training? I cannot train the network for more than about 10 epochs as the loss becomes NaN by then.

**Comment by zhangshuaitao** [comment]
You can try increasing the batchsize.

**Comment by GuangtaoLyu** [comment]
> You can try increasing the batchsize.

I increase the batchsize from 4 to 10. the loss still becomes NaN.

**Comment by UsamaRehman001** [comment]
> Are the default settings of parameters such as learning rate the parameters that you used for training? I cannot train the network for more than about 10 epochs as the loss becomes NaN by then.

Hey you sorted out your issue?
